nucleus/src/main/java/za/co/absa/subatomic/infrastructure/team/view/jpa/TeamPersistenceHandler.java

```diff
@@ -267,6 +291,58 @@ public TeamEntity updateOpenShiftCloud(TeamEntity team,
         return this.teamRepository.save(team);
     }
 
+    @Transactional
+    public TeamEntity updateTeamMetadata(TeamEntity team, List<MetadataResource> metadata, boolean overwrite) {
```

Comment:
Can we break this function up into two functions because this is quite big. Mimicking the two from service seems like a good idea to me: setTeamMetadata and updateTeamMetadata

Reply:
Owatta. Domo arigato sen pai